### ISSUE TITLE
ERROR log:  Unable to prepare setup for platform thermiagenesis.climate: Platform not found (No module named 'homeassistant.util.temperature')

### ISSUE THREAD

**Issue by HenrikBurton** [CJNE/thermiagenesis#238]
## Version of the custom_component

Problem appered with HA 2023.11.
Current installed versions are:
Core 2023.11.2
Supervisor 2023.11.0
Operating System 11.1
Frontend 20231030.2

## Configuration

Heater: Thermia Calibra 12
Heatpump model*: Inverter

## Describe the bug

After upgrading to 2023.11 this log item appeared and the thermostat entity for Tap water and Heating disappeared. Rollback to 2023.10.5 solve the problem.

## Debug log

2023-11-12 10:31:59.893 DEBUG (MainThread) [custom_components.thermiagenesis] Finished fetching thermiagenesis data in 0.002 seconds (success: True)
**2023-11-12 10:31:59.908 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform thermiagenesis.climate: Platform not found (No module named 'homeassistant.util.temperature').**
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_first_prioritised_demand
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_compressor_speed_rpm
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_compressor_current_gear
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_queued_demand_second_priority
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_queued_demand_third_priority
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_brine_in_temperature
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_brine_out_temperature
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_compressor_operating_hours
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_tap_water_operating_hours
2023-11-12 10:32:00.461 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: input_external_additional_heater_operating_hours
2023-11-12 10:32:00.470 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: coil_enable_tap_water
2023-11-12 10:32:00.470 INFO (MainThread) [custom_components.thermiagenesis] Register attribute for update: coil_enable_heat

**Comment by dd1982** [comment]
> Sorry, my bad. Forgott that I had to update the unit myself. Works now after update 🤩

What exactly did you do? Have the same issue - climate entities disappeared after updating Home Assistant to 2023.11 and I get "Unable to prepare setup for platform thermiagenesis.climate: Platform not found (No module named 'homeassistant.util.temperature')." in the logs.

**Comment by HenrikBurton** [comment]
> > Sorry, my bad. Forgott that I had to update the unit myself. Works now after update 🤩
> 
> What exactly did you do? Have the same issue - climate entities disappeared after updating Home Assistant to 2023.11 and I get "Unable to prepare setup for platform thermiagenesis.climate: Platform not found (No module named 'homeassistant.util.temperature')." in the logs.

I followed the installation [instruction](https://github.com/CJNE/thermiagenesis#installation) as if it was the first installation and restarted HA, voila. In addition, some entities had got other ID's (sensor to number) so you might have to change your configuration and panels a bit.

**Comment by Lord-Bam** [comment]
@kejXero 

1. I removed the direcory, in /homeassistant/custom_components/thermiagenesis
2. Restarted home assistant.
3. Created the directory /homeassistant/custom_components/thermiagenesis again.
4. Copied the files from /thermiagenesis-master/custom_components/thermiagenesis to the created directory.
5. Restarted home assistant.
6. Added and configured the integration "Thermia Genesis"
7. Maybe restarted home assistant... 

That was it.
